packages/patternfly-4/react-core/src/components/AboutModal/AboutModalBoxContent.d.ts

```diff
@@ -1,11 +1,11 @@
 import { FunctionComponent, HTMLProps, ReactNode } from 'react';
 
-export interface ModalBoxBodyProps extends HTMLProps<HTMLDivElement> {
+export interface AboutModalBoxContentProps extends HTMLProps<HTMLDivElement> {
   children: ReactNode;
-  tradeMark: string;
+  trademark: string;
   id: string;
 }
 
-declare const ModalBoxBody: FunctionComponent<ModalBoxBodyProps>;
```

Comment:
Noticed a lot of the other d.ts files are not correct... Where you just ignoring the interfaces before?  How are you using this at the moment in typescript?

Reply:
We're using AboutModal in a .jsx file in OpenShift. I talked to @dgutride about these weird d.ts files and he said since they're not being exported, they're probably not breaking things yet.

Reply:
Thanks!